websites/A/AnizmTV/presence.ts

```diff
@@ -0,0 +1,134 @@
+const presence = new Presence({
+  clientId: "778715860638367804"
+});
+
+var strings = presence.getStrings({
+  playing: "presence.playback.playing",
+  paused: "presence.playback.paused",
+  browsing: "presence.activity.browsing"
+});
+
+function getTimestamps(
+  videoTime: number,
+  videoDuration: number
+): Array<number> {
+  let startTime = Date.now();
+  let endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+  return [Math.floor(startTime / 1000), endTime];
+}
+
+const startTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement,
+tags: HTMLElement;
+
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
+  if (!msg) return;
+  video = msg;  
+});
+
+presence.on("UpdateData", async () => {
+  
+  const data: PresenceData = {
+    largeImageKey: "anizm"
+  };
+
+  var title = document.querySelector(
+    "html > body > font > main > #pageContent > div > h2 > a"
+  );
+  var episode = document.querySelector(
```

Comment:
merge consts

Reply:
http://prntscr.com/vmyecj
Bot sees wrong

Reply:
no, it doesn't, please fix codacy errors.

Reply:
I'm sorry, it's my fault.